docs/devel/issues.md

```diff
@@ -33,23 +33,21 @@ Documentation for other releases can be found at
 GitHub Issues for the Kubernetes Project
 ========================================
 
-A list quick overview of how we will review and prioritize incoming issues at https://github.com/kubernetes/kubernetes/issues
+A quick overview of how we will review and prioritize incoming issues at https://github.com/kubernetes/kubernetes/issues
 
 Priorities
 ----------
 
-We will use GitHub issue labels for prioritization.  The absence of a priority label means the bug has not been reviewed and prioritized yet.
+We use GitHub issue labels for prioritization.  The absence of a
+priority label means the bug has not been reviewed and prioritized
```

Comment:
Can you clarify this a bit - we've changed it slightly since the doc was originally written.
I would say: "The absence of any label means it has not been reviewed by a human yet. The presence of a team label but not a priority label means it has been sorted by the oncall but not yet triaged by the responsible team. The presence of a team and priority label means it has been triaged by the responsible team."

(or something to that effect)